include/wayfire/config/option.hpp

```diff
@@ -155,6 +158,19 @@ class option_t : public option_base_t,
         : option_base_t(name), default_value(def_value), value(default_value)
     { }
 
+    /**
+     * Create a copy of the option.
+     */
+    virtual std::shared_ptr<option_base_t> clone_option() const override
+    {
+        auto result = std::make_shared<option_t>(get_name(), get_default_value());
+        if constexpr (std::is_arithmetic<Type>::value) {
```

Comment:
If clonse is really supposed to clone the option, then you should also copy the current value of the option.